docs/source/using/troubleshooting.adoc

```diff
@@ -243,3 +243,51 @@ Workaround: Avoid installing packages or storing large files in the root filesys
 Instead, you can create a sub-directory in the *_mnt/sda1/_* persistent storage volume or define and mount xref:../using/host-folders.adoc#[host folders] that can share storage space between the host and the Minishift VM.
 
 If you want to perform development tasks inside the Minishift VM, it is recommended that you use containers, which are stored in persistent storage volumes, and reuse the xref:../using/docker-daemon.adoc#[Minishift Docker daemon].
+
+
+[[minshift-start-failed-due-to-startup-check]]
+== Minishift startup check failed
+
+During the start of Minishift we perform several pre-flight checks to make sure the Minishift VM and OpenShift Cluster are able to start without any issues. Therefore, if something is not correctly configured, we fail a test and the startup will stop.
+
+If the startup failed for one of the driver plugins, please make sure they are correctly configured by checking the  xref:../getting-started/setting-up-driver-plugin.adoc#[setting up the driver plugin] topic. If for some reason you want to force the startup, you can configure Minishift to treat these errors as warnings as follows:
+
+. For KVM/Libvirt on Linux, run the following command:
+
```

Comment:
You need a '+' character here to make the listing part of the enumeration. Numbering as well as indentation are off now. Please review the docs in rendered form as well.